include/kernels/madwf_transfer.cuh

```diff
@@ -0,0 +1,150 @@
+#pragma once
+
+#include <color_spinor_field.h>
+#include <color_spinor_field_order.h>
+#include <shared_memory_cache_helper.cuh>
+#include <madwf_transfer.h>
+
+namespace quda
+{
```

Comment:
Some doxygen in this file would be good.  As would a brief overall description of what this kernel does.

Reply:
https://github.com/lattice/quda/pull/1061/commits/cefb2a527fa7cb1303a9e7487b9fea4f39d1bf8d